server/controllers/finance/reports/financial.employee.handlebars

```diff
@@ -29,8 +40,11 @@
             <td class="text-right" title="{{this.trans_id}}">{{this.trans_id}}</td>
             <td class="text-uppercase" style="max-width : 200px; white-space : nowrap; overflow : hidden; text-overflow : ellipsis;" title="{{this.description}}">{{this.description}}</td>
             <td class="text-right">
-              {{debcred this.balance ../metadata.enterprise.currency_id}}
+              {{debcred this.debit ../metadata.enterprise.currency_id}}
```

Comment:
This should just be
```hbs
{{ currency this.debit ../metadata.enterprise.currency_id }}
```
Same with the `this.credit` one as well.  We only need to show debits/credits in different colors if they are in the same column.